scripts/run_slurm.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#SBATCH --job-name=parallel_data_processing
+#SBATCH --ntasks=
+#SBATCH --nodes=
+#SBATCH --time=
+#SBATCH --partition=your_partition_name
+#SBATCH --output=parallel_data_processing_%j.out
+#SBATCH --error=parallel_data_processing_%j.err
+#SBATCH --exclusive
+
+# set data-juicer and config file path
+datajuicer_path=  # 替换为 data-juicer 的实际路径
```

Comment:
Better to add English comments aligned with other comments.

There are two more similar situations below on line 13, 23.

Reply:
Resolved